requirements.txt

```diff
@@ -68,7 +68,7 @@ pycparser==2.18
 pycrypto==2.6.1
 pyflakes==2.0.0
 Pygments==2.2.0
-pytest==3.6.2
```

Comment:
Can you revert this change? [Paul found that the newer version breaks our tests](https://github.com/uwcirg/true_nth_usa_portal/pull/2397)